hexrdgui/llnl_import_tool_dialog.py

```diff
@@ -540,20 +575,22 @@ def load_detector_images(self):
                 self.ui.accept_detector,
                 enabled=bool(selected['data'] and selected['dark']))
 
-    def load_images(self):
+    def load_images(self, checked=False, selected_file=None):
```

Comment:
Is this `checked` argument used for anything?

Reply:
This is also connected to the [self.ui.image_plate_load.clicked](https://github.com/HEXRD/hexrdgui/pull/1818/files#diff-ac087ac85102dd9829f7addaa727a770a24b4a510dd23ec4b6faf36cfe9fccc8R114) signal, which passes in a `checked` argument.